src/stream/mod.rs

```diff
@@ -112,7 +112,7 @@ if_std! {
     pub use self::collect::Collect;
     pub use self::wait::Wait;
     pub use self::split::{SplitStream, SplitSink};
-    pub use self::futures_unordered::{futures_unordered, FuturesUnordered};
+    pub use self::futures_unordered::{futures_unordered, FuturesUnordered, FuturesUnorderedIterMut};
```

Comment:
I probably wouldn't re-export this type. Instead, maybe make the `futures_unordered` module pub (and ensure no priv types are accidentally exported).